<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-BL99EMON" failures="1" tests="1" name="week3.day1.LearnAssertions" time="11.968" errors="0" timestamp="2021-12-18T11:24:31 IST" skipped="0">
  <testcase classname="week3.day1.LearnAssertions" name="createIncident" time="11.968">
    <failure message="1 expectation failed.
JSON path result doesn&#039;t match.
Expected: a collection containing &quot;sys_updated_on&quot;
  Actual: &lt;{parent=, made_sla=true, caused_by=, watch_list=, upon_reject=cancel, sys_updated_on=2021-12-18 05:54:28, child_incidents=0, hold_reason=, task_effective_number=INC0010068, approval_history=, number=INC0010068, resolved_by=, sys_updated_by=admin, opened_by={link=https://dev62655.service-now.com/api/now/table/sys_user/6816f79cc0a8016401c5a33be04be441, value=6816f79cc0a8016401c5a33be04be441}, user_input=, sys_created_on=2021-12-18 05:54:28, sys_domain={link=https://dev62655.service-now.com/api/now/table/sys_user_group/global, value=global}, state=1, route_reason=, sys_created_by=admin, knowledge=false, order=, calendar_stc=, closed_at=, cmdb_ci=, delivery_plan=, contract=, impact=3, active=true, work_notes_list=, business_service=, priority=5, sys_domain_path=/, rfc=, time_worked=, expected_start=, opened_at=2021-12-18 05:54:28, business_duration=, group_list=, work_end=, caller_id=, reopened_time=, resolved_at=, approval_set=, subcategory=, work_notes=, universal_request=, short_description=Assertion, close_code=, correlation_display=, delivery_task=, work_start=, assignment_group=, additional_assignee_list=, business_stc=, description=created from ppostman, calendar_duration=, close_notes=, notify=1, service_offering=, sys_class_name=incident, closed_by=, follow_up=, parent_incident=, sys_id=5fdec17c2f200110fa64ad2ef699b614, contact_type=, reopened_by=, incident_state=1, urgency=3, problem_id=, company=, reassignment_count=0, activity_due=, assigned_to=, severity=3, comments=, approval=not requested, sla_due=, comments_and_work_notes=, due_date=, sys_mod_count=0, reopen_count=0, sys_tags=, escalation=0, upon_approval=proceed, correlation_id=, location=, category=inquiry}&gt;
" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 1 expectation failed.
JSON path result doesn't match.
Expected: a collection containing "sys_updated_on"
  Actual: <{parent=, made_sla=true, caused_by=, watch_list=, upon_reject=cancel, sys_updated_on=2021-12-18 05:54:28, child_incidents=0, hold_reason=, task_effective_number=INC0010068, approval_history=, number=INC0010068, resolved_by=, sys_updated_by=admin, opened_by={link=https://dev62655.service-now.com/api/now/table/sys_user/6816f79cc0a8016401c5a33be04be441, value=6816f79cc0a8016401c5a33be04be441}, user_input=, sys_created_on=2021-12-18 05:54:28, sys_domain={link=https://dev62655.service-now.com/api/now/table/sys_user_group/global, value=global}, state=1, route_reason=, sys_created_by=admin, knowledge=false, order=, calendar_stc=, closed_at=, cmdb_ci=, delivery_plan=, contract=, impact=3, active=true, work_notes_list=, business_service=, priority=5, sys_domain_path=/, rfc=, time_worked=, expected_start=, opened_at=2021-12-18 05:54:28, business_duration=, group_list=, work_end=, caller_id=, reopened_time=, resolved_at=, approval_set=, subcategory=, work_notes=, universal_request=, short_description=Assertion, close_code=, correlation_display=, delivery_task=, work_start=, assignment_group=, additional_assignee_list=, business_stc=, description=created from ppostman, calendar_duration=, close_notes=, notify=1, service_offering=, sys_class_name=incident, closed_by=, follow_up=, parent_incident=, sys_id=5fdec17c2f200110fa64ad2ef699b614, contact_type=, reopened_by=, incident_state=1, urgency=3, problem_id=, company=, reassignment_count=0, activity_due=, assigned_to=, severity=3, comments=, approval=not requested, sla_due=, comments_and_work_notes=, due_date=, sys_mod_count=0, reopen_count=0, sys_tags=, escalation=0, upon_approval=proceed, correlation_id=, location=, category=inquiry}>

at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:72)
at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:105)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:493)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:674)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
at io.restassured.internal.ResponseSpecificationImpl.body(ResponseSpecificationImpl.groovy:261)
at io.restassured.specification.ResponseSpecification$body$0.callCurrent(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:51)
at io.restassured.specification.ResponseSpecification$body$0.callCurrent(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.body(ResponseSpecificationImpl.groovy:108)
at io.restassured.internal.ValidatableResponseOptionsImpl.body(ValidatableResponseOptionsImpl.java:244)
at io.restassured.internal.ValidatableResponseImpl.super$2$body(ValidatableResponseImpl.groovy)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)
at org.codehaus.groovy.runtime.ScriptBytecodeAdapter.invokeMethodOnSuperN(ScriptBytecodeAdapter.java:144)
at io.restassured.internal.ValidatableResponseImpl.body(ValidatableResponseImpl.groovy:298)
at io.restassured.internal.ValidatableResponseImpl.body(ValidatableResponseImpl.groovy)
at week3.day1.LearnAssertions.createIncident(LearnAssertions.java:48)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- createIncident -->
  <system-out/>
</testsuite> <!-- week3.day1.LearnAssertions -->
